### ISSUE TITLE
Clear git history of gifs

### ISSUE THREAD

**Issue by felixfung** [felixfung/skippy-xd#110]
Previously I uploaded gifs `gifs/switch.gif`, `gifs/expose.gif`, `gifs/paging.gif` into the repo, which is ~50M in size, unnecessarily clogging network when cloning, albeit far from crippling.

I have since transferred these gifs to https://github.com/felixfung/skippy-xd-gifs.

As last step of the operation, I have to remove the gifs in this repo, not just with git rm gifs/*.gif, as the gifs would still reside in the git history. Rather, I did

```
python git-filter-repo --invert-paths --path gifs/switch.gif
python git-filter-repo --invert-paths --path gifs/expose.gif
python git-filter-repo --invert-paths --path gifs/paging.gif
```

```
python  git-filter-repo --strip-blobs-bigger-than 1M
```

in PR #107, #108, #109.

These did NOT solve the problem. The reason was probably because when these operations are performed via PR, merging would preserve the original branch and its history, and hence the gifs in the history.

I have since done 
```
python  git-filter-repo --strip-blobs-bigger-than 1M
```
and force pushed to master.

When doing a fresh git clone, the repo size is around 50M, indicating the gifs are still in git history.

As experiment, I have pushed this repo to https://github.com/felixfung/skippy-clone, so that it should have exact content as well as git history. When cloned, it has repo size of around 1M, indicating the gifs are complete cleaned from git history.

The only difference between https://github.com/felixfung/skippy-xd and https://github.com/felixfung/skippy-clone is that one is tracked as github fork and one is not. It seems to me github fork is potentially tracking history between forks, hence the history of big binaries are not removed.

In summary, I have successfully reduced the repo footprint to minimum size, with no adverse effect.

TL;DR: the repo footprint is halved. For those who want minimal repo footprint, you can clone from https://github.com/felixfung/skippy-clone.
